OpenRA.Mods.Common/Traits/SupportPowers/ProduceActorPower.cs

```diff
@@ -28,15 +28,21 @@ public class ProduceActorPowerInfo : SupportPowerInfo
 		public readonly string Type = null;
 
 		[NotificationReference("Speech")]
-		[Desc("Notification played when production is activated.",
+		[Desc("Speech notification played when production is activated.",
 			"The filename of the audio is defined per faction in notifications.yaml.")]
 		public readonly string ReadyAudio = null;
 
+		[Desc("Text notification displayed when production is activated.")]
+		public readonly string ReadyTextNotification = null;
```

Comment:
Thinking out loud: Would we want to support notifications of the form "Unit Ready: Mammoth Tank", "Unit Lost: Minigunner", etc? If yes, we will need to consider how to expose the format placeholders to yaml... the Fluent translation system provides an elegant solution, so perhaps we should aim to advance our translation migration first and put all of these straight into the ftl files?

Reply:
It seems like a nice feature on paper but I'm not convinced we should put it in the default mods. For one - the speech notifications just say "Unit lost" so one can argue that a player with text notifications turned on has an advantage.

And I thought we're using string IDs to reference translations? So the placeholders live in the fluent translation files? :thinking:

Reply:
Ok. I don't want to derail getting this merged so happy to defer this to a future discussion.

About the format placeholders, I mean that with the current system we're forced to define strings in yaml like `Unit Lost: {0}`, whereas the equivalent in Fluent would be `notification-unitlost = Unit Lost: { $actor-name }`.